src/plugins/data/public/actions/filters/create_filters_from_value_click.test.ts

```diff
@@ -25,16 +25,18 @@ import {
 } from '../../../public';
 import { dataPluginMock } from '../../../public/mocks';
 import { setIndexPatterns, setSearchService } from '../../../public/services';
-import { createFiltersFromValueClickAction } from './create_filters_from_value_click';
-import { ValueClickContext } from '../../../../embeddable/public';
+import {
+  createFiltersFromValueClickAction,
+  ValueClickDataContext,
+} from './create_filters_from_value_click';
 
 const mockField = {
   name: 'bytes',
   filterable: true,
 };
 
 describe('createFiltersFromValueClick', () => {
-  let dataPoints: ValueClickContext['data']['data'];
```

Comment:
Rather than selecting the `data` property from `ValueClickContext`, I've copied those pieces over here.